docusaurus.config.js

```diff
@@ -13,6 +13,13 @@ module.exports = {
   projectName: 'docs', // the github project name. Will be used in the deploy step to clone the repository
   themeConfig: {
     image: 'img/expert_dark_preview.png',
+    announcementBar: {
+      id: 'sdk-refactor-announcement',
+      content: 'Our javascript sdk has received a major overhaul with the latest version 0.29.0! Check out the <a target="_blank" href="https://github.com/KILTprotocol/sdk-js/releases/tag/0.29.0">release notes</a> to find out what changed. Check what to consider when upgrading <a href="/docs/develop/sdk/cookbook/upgrading_to_v0_29/">here</a>.',
```

Comment:
Can we use markdown instead? Or is it not possible? In that case we could detect if and when the link gets broken.

```suggestion
      content: 'Our Javascript SDK has undergone a major overhaul with the latest version 0.29.0! Check out the <a target="_blank" href="https://github.com/KILTprotocol/sdk-js/releases/tag/0.29.0">release notes</a> to find out what changed. Check what to consider when upgrading <a href="/docs/develop/sdk/cookbook/upgrading_to_v0_29/">here</a>.',
```

Reply:
it doesn't take markdown unfortunately